docs/Text.md

```diff
@@ -90,9 +90,12 @@ When `split_only == True`, returns `txt` split into lines in an array (with any
 ## .write()
 Prints multi-line text between the page margins, starting from the current position.
 When the right margin is reached, a line break occurs at the most recent
-space or soft-hyphen character, and text continues from the left margin.
-A manual break happens any time the \\n character is met,
-Upon method exit, the current position is left near the end of the text, ready for the next call to continue without a gap, potentially with a different font or size set. Returns a boolean indicating if page break was triggered.
+space or soft-hyphen character (in word wrap mode) or at the current position (in
+character break mode), and text continues from the left margin.
```

Comment:
Similarly to the "Right-to-Left & Arabic Script workaround" section above,
maybe an explicit suggestion could be made of using `wrapmode='CHAR'` for langages like chinese or japanese that do not separate words with spaces?

Reply:
The whole section is about limitations (and their workarounds) right now, which this really isn't.
But we can turn it into a general typography and language specifics section.

Reply:
> But we can turn it into a general typography and language specifics section.

Yes, this would be great!